docs/releases/2024-january.md

```diff
@@ -14,5 +14,20 @@ nav_order: 6
 GlareDB now supports ClickHouse as an external data source. Learn more in our
 [ClickHouse docs](/docs/data-sources/supported/clickhouse).
 
+## Table functions
+
+**Available in**: [GlareDB@v0.8.0], [GlareDB Cloud]
+
+- `read_sqlserver` to read tables from SQL Server without adding it as a data
+  sources:
```

Comment:
request: there's a typo here (sources -> source)  _however_

suggestion:

- `read_sqlserver` to read tables from SQL Server. The tables do not have to be a known data source to
GlareDB.


the rationale here is "without adding it as a data source" is ambiguous (is it _only_ to be used for non data source?)

Reply:
will remove that part.